### PR TITLE
docs/corobo.rst: Document corobo commands

### PR DESCRIPTION
**Pull request by meetmangukiya**
# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: docs/corobo.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Using corobo
+============
+
+GitHub/GitLab utilities
+-----------------------
+
+1. Getting invitation to the org:
+   Send ``helloworld`` in the chat room.
```
Comment (SanketDG) [inline docs/corobo.rst]: just helloworld? If we support "Hello World", better to mention that

Comment (SanketDG) [inline docs/corobo.rst]: does this work with gitlab?

File: docs/corobo.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+   Add ``pending_review`` label:
+   ``corobo mark pending <pr-link>``
+
+coala utilities
+---------------
+
+1. Wanna know stats regarding coala?:
+
+   a. Get user contribution stats using:
+      ``corobo contrib stats meetmangukiya``
```
Comment (SanketDG) [inline docs/corobo.rst]: You may want to mention the user part explicitly

File: docs/corobo.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+   Send ``helloworld`` in the chat room.
+2. Want to work on an issue, first assign yourself:
+   ``corobo assign <issue-link>``
+3. Assigned issue is time-consuming, no longer interesting? Wanna discontinue:
+   ``corobo unassign <issue-link>``
+4. Wanna file a new issue, but don't want to leave gitter?:
+   ``corobo new issue <repo> <title>\n[description\n]+``
+5. PR is up, waiting for a review. Consider reviewing other PRs while you wait
+   for review of your PR. If the PR needs more work, mark it
+   work-in-progress(wip):
```
Comment (SanketDG) [inline docs/corobo.rst]: This whole paragraph could be confusing for newcomers. Just mention about marking a PR as wip

File: docs/corobo.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+1. Someone is asking a googlable question? Respond with a lmgtfy:
+   ``corobo lmgtfy this is the search string``
+2. Get bot to apologize:
+   ``corobo nm``
+3. Explain pre-defined terms:
+   ``corobo explain rebase``
+4. Is your PR ready to merge, go shipit!
+   ``corobo ship it``
+5. Ask a question to knowledge engine:
```
Comment (SanketDG) [inline docs/corobo.rst]: knowledge engine?

File: docs/corobo.rst (hunk 1 above)
Comment (meetmangukiya) [inline docs/corobo.rst]: no, we only invite people on github org, gitlab invites are exclusive

File: docs/corobo.rst (hunk 2 above)
Comment (meetmangukiya) [inline docs/corobo.rst]: alright :sweat_smile:

File: docs/corobo.rst (hunk 4 above)
Comment (meetmangukiya) [inline docs/corobo.rst]: forgot to mention `wolfram`

File: docs/corobo.rst (hunk 1 above)
Comment (SanketDG) [inline docs/corobo.rst]: better to mention that then

**Comment by Udayan12167** [comment]
I think each point needs a bit of description of what it returns. Like what do Bear stats return or PR stats return or Language stats return

File: docs/corobo.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+      ```
+      import this
+      this = 3
+      some = 'code'
+      ```
+
+Answer
+------
+
+You have got a question, no one's available to answer. Let bot try to answer it
```
Comment (Udayan12167) [inline docs/corobo.rst]: *Let the bot try to answer it for you

File: docs/corobo.rst [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+for you:
+``corobo answer your question string goes here``
+
```
Comment (Udayan12167) [inline docs/corobo.rst]: Also I this is a good place to describe what the bot can and cannot answer

File: docs/corobo.rst (hunk 5 above)
Comment (Udayan12167) [inline docs/corobo.rst]: Or Let corobo try to...

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/269/commits/684350c4c5881c2dc2079f7fdd6a8c766f00fd80

**Comment by meetmangukiya** [comment]
ack 2f99c2b

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**